docs/langgraph/README.md

```diff
@@ -0,0 +1,46 @@
+# LangGraph.js Complete Guide
+
+This directory contains a comprehensive guide for LangGraph.js - a low-level orchestration framework for building controllable agents.
+
+## 📁 Documentation Structure
+
+### Getting Started
+- **[installation.md](installation.md)** - Installation and setup
+
+### Core Concepts
+- **[core-concepts.md](core-concepts.md)** - Graph state definition, basic construction, and node patterns
+- **[control-flow.md](control-flow.md)** - Branching, map-reduce, conditional routing, and loop control
+- **[state-management.md](state-management.md)** - State schema design, updates, and persistence patterns
+
+### Advanced Features
+- **[persistence-memory.md](persistence-memory.md)** - Thread-level persistence, memory management, and subgraph persistence
+- **[human-in-the-loop.md](human-in-the-loop.md)** - Interrupts, tool call review, breakpoints, and time travel
+- **[streaming.md](streaming.md)** - Stream modes, token-level streaming, and custom data streaming
+- **[tool-calling.md](tool-calling.md)** - ToolNode usage, patterns, and state updates from tools
+- **[multi-agent.md](multi-agent.md)** - Agent communication, supervisory patterns, and multi-turn conversations
+
+### Production & Platform
+- **[advanced-features.md](advanced-features.md)** - Subgraphs, retries, caching, runtime configuration, and structured output
+- **[prebuilt-agents.md](prebuilt-agents.md)** - ReAct agents and custom agent patterns
+- **[platform.md](platform.md)** - LangGraph Platform deployment, authentication, assistants, and frontend integration
```

Comment:
Since we are not using the LangGraph Platform, it seems best not to include it.

Reply:
I have also deleted the documentation related to the platform!
https://github.com/liam-hq/liam/pull/3486/commits/f2a56672101574079989f9ecb11a81e08ca6fdb3